git-hooks/pre-commit

```diff
@@ -7,7 +7,7 @@ const execa = require('execa');
 const chalk = require('chalk');
 
 execa
-    .stdout('git', ['diff', '--name-only', '--cached'])
+    .stdout('git', ['diff', '--name-only', '--cached', '--diff-filter=AM'])
```

Comment:
How so? Without that line, when I merged master after a long time and had a long list of deletions with a conflict, and I fixed the conflict and committed, the lint would fail because of the deletions.